### PR TITLE
Update get_tpm_manufacturer function return type

### PR DESCRIPTION
**Pull request by leonjia0112**
1. Changed return type to Result<> instead of Option<>.
2. Removed .unwrap() calls in this function.
3. Return the manufacturer name once the token is found instead of returning at the end of the function.

**Test passed. Ready for a review.**
Thanks!

### CODE REVIEW

File: src/tpm.rs [hunk 1]
```diff
@@ -158,33 +158,34 @@ pub fn is_vtpm() -> Option<bool> {
...
         true,
         false,
         String::new(),
     );
-    let content_result = String::from_utf8(return_output);
-    let content = content_result.unwrap();
+
+    let content = match String::from_utf8(return_output) {
+        Ok(c) => c,
+        Err(e) => return emsg("Faield to convert output to String.", Some(e)),
```
Comment (lukehinds) [inline src/tpm.rs]: Small typo /s/Faield/Failed

```suggestion
         Err(e) => return emsg("Failed to convert output to String.", Some(e)),
```

Comment (lukehinds) [reply src/tpm.rs]: No worries, that's about the best I can offer for now on rust reviews :)

**Comment by leonjia0112** [comment]
Changed to fix the typo error.

**Test passed. Ready for a review.**
Thanks!

**Comment by lukehinds** [comment]
If you can fix up the conflicts @leonjia0112 , I think this will be ready to merge as @frozencemetery gives it a +1 here: https://github.com/keylime/rust-keylime/pull/35#pullrequestreview-187501697

**Pull request closed without merging**